src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php

```diff
@@ -41,7 +41,7 @@
 
                 $sTmpFormTabsContent .= '
                 <div class="form-group table-editor-form-group row" id="fieldname_'.TGlobal::OutHTML($oField->name).'">
-                    <div class="col-sm-2 col-form-label">
```

Comment:
This padding is the only thing that "col-form-label" does?

Reply:
I used the bootstrap class "col-form-lable" again and overwrote the paddings so it aligns with the standard-fields.